damus/Models/UserSettings.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  UserSettings.swift
+//  damus
+//
+//  Created by Suhail Saqan on 12/29/22.
+//
+
+import Foundation
+
+struct WalletItem : Decodable, Identifiable, Hashable {
+    var id: Int
+    var tag: String
+    var name : String
+    var link : String
+    var appStoreLink : String
+    var image: String
+}
+
+// New url prefixes needed to be added to LSApplicationQueriesSchemes
+enum Wallet: String, CaseIterable {
+    case defaultwallet = """
```

Comment:
this name is a bit confusing now, given that the _user_ can have a default wallet and the _system_ can have a default lightning wallet.

It's probably would be best to change it to `systemDefaultLightningWallet` to be more explicit, and differentiate it from the user-selected default wallet.